scripts/gha/get_pr_review_comments.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python3
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Fetches and formats review comments from a GitHub Pull Request."""
+
+import argparse
+import os
+import sys
+import firebase_github
+import datetime
+from datetime import timezone, timedelta
+
+
+# Attempt to configure logging for firebase_github if absl is available
+try:
+    from absl import logging as absl_logging
+    # Set verbosity for absl logging if you want to see logs from firebase_github
+    # absl_logging.set_verbosity(absl_logging.INFO)
+except ImportError:
+    pass # firebase_github.py uses absl.logging.info, so this won't redirect.
+
+def main():
+    default_owner = firebase_github.OWNER
+    default_repo = firebase_github.REPO
+
+    parser = argparse.ArgumentParser(
+        description="Fetch review comments from a GitHub PR and format into simple text output.",
+        formatter_class=argparse.RawTextHelpFormatter
+    )
+    parser.add_argument(
+        "--pull_number",
+        type=int,
+        required=True,
+        help="Pull request number."
+    )
+    parser.add_argument(
+        "--owner",
+        type=str,
+        default=default_owner,
+        help=f"Repository owner. Defaults to '{default_owner}'."
+    )
+    parser.add_argument(
+        "--repo",
+        type=str,
+        default=default_repo,
+        help=f"Repository name. Defaults to '{default_repo}'."
+    )
+    parser.add_argument(
+        "--token",
+        type=str,
+        default=os.environ.get("GITHUB_TOKEN"),
+        help="GitHub token. Can also be set via GITHUB_TOKEN env var."
+    )
+    parser.add_argument(
+        "--context-lines",
+        type=int,
+        default=0,
```

Comment:
OK, let's set this default back to 10, but be a little bit smarter and still display the FIRST line of the hunk if it starts with "@@". This is because there can be line and function name context here that could be useful. You can print a "..." to imply that there is skipped content.